frontend/src/components/pages/rp-connect/secrets/Secrets.Create.tsx

```diff
@@ -0,0 +1,142 @@
+import {Button, ButtonGroup, createStandaloneToast, Flex, FormField, Input, PasswordInput} from '@redpanda-data/ui';
+import {PageComponent, PageInitHelper} from '../../Page';
+import {observer} from 'mobx-react';
+import {appGlobal} from '../../../../state/appGlobal';
+import {pipelinesApi, rpcnSecretManagerApi} from '../../../../state/backendApi';
+import PageContent from '../../../misc/PageContent';
+import {action, makeObservable, observable} from 'mobx';
+import {DefaultSkeleton} from '../../../../utils/tsxUtils';
+import {formatPipelineError} from '../errors';
+import {CreateSecretRequest, Scope} from '../../../../protogen/redpanda/api/dataplane/v1alpha2/secret_pb';
+
+const {ToastContainer, toast} = createStandaloneToast();
+
+const returnSecretTab = '/connect-clusters?defaultTab=redpanda-connect-secret'
+
+
+@observer
+class RpConnectSecretCreate extends PageComponent {
```

Comment:
When we try to load these new pages and RP-connect is not enabled, we get infinite loading, this applies for both new URLs that have been added to the router. Can we please show something that prompts the user to configure it?

Reply:
when it happens, rpcn-secret continue working:

https://redpandadata.slack.com/archives/C074X8HD68L/p1732568383289989?thread_ts=1732294471.397329&cid=C074X8HD68L